templates/install-wp-tests.sh

```diff
@@ -14,6 +14,12 @@ WP_VERSION=${5-latest}
 WP_TESTS_DIR=${WP_TESTS_DIR-/tmp/wordpress-tests-lib}
 WP_CORE_DIR=${WP_CORE_DIR-/tmp/wordpress/}
 
+if [[ $WP_VERSION =~ [0-9]+\.[0-9]+(\.[0-9]+)? ]]; then
+	WP_TESTS_TAG="tags/$WP_VERSION"
+else
+	WP_TESTS_TAG='trunk'
```

Comment:
I have also felt the same problem.
Sorry, but not know how to tie the version number and the "latest".

In addition, in the case of plugins, "Tested up to:" is in the Readme.txt.
So I think should be tested in the explicit version. What do you think?